commons/memcached/1.5.20/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM cloudsuite/debian:base-os
```

Comment:
Could you please update the memcached image to the latest version?

Reply:
This can be helpful: https://github.com/docker-library/memcached/blob/4538128227a0e422e59df735d67b03ee23d39637/debian/Dockerfile

Reply:
Updated